### PR TITLE
Insta crawler

### PR DESCRIPTION
**Pull request by priyank200**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.env
+data
```
Comment (sgdanawe) [inline .gitignore:5]: Add a new line at the end.

File: instagram.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        file.close()
+        count = count + 1
+    return follow_list
+
+
+if __name__ == '__main__':
+    # logger.info("Getting followers of cristiano")
+    # follower_list = scrape_followers("cristiano")  # calling the function to scrap data
+    # logger.info("Done")
+    Follower_data = pd.read_csv("data/Instagram/data.csv")
```
Comment (sgdanawe) [inline instagram.py]: Variable name should be lower-case

File: main.py [hunk 3]
```diff
@@ -1,133 +0,0 @@
...
-try:
-    data("cristiano") #calling the function to scrap data
-except Exception as e:
-    print(e)
-
-
-list = pd.read_csv("names.csv")
-# print(list["name"])
-for name in list["name"]:
-    Username_scrap = name
```
Comment (sgdanawe) [inline main.py:131]: Variable name

File: main.py [hunk 4]
```diff
@@ -1,133 +0,0 @@
...
-        print(e)
-        print(f"ERROR for {Username_scrap}")
-
-#Cfunction to collect username for scraping the data
-def data(name):
-    instance = instaloader.Instaloader(save_metadata=False, compress_json=False, )
-    YOUR_USERNAME = "tanishsharma869"
-    YOUR_PASSWORD = "qwerty54321"
-
-    Username_scrap = name
```
Comment (sgdanawe) [inline main.py:108]: Variable Name

File: instagram.py [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    # print(Follower_data["username"][0])
+    follower_list = list(Follower_data["username"])
+    count = 0
+    for name in follower_list:
+        logger.info(f"Scrapping {name}")
+        insta_scrapper(name)
+        logger.info(f"Scraped {name}")
+        Follower_data = Follower_data.drop([count])
```
Comment (sgdanawe) [inline instagram.py]: Here too

**Review by sgdanawe — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (priyank200) [reply .gitignore:5]: Done

File: instagram.py (hunk 2 above)
Comment (priyank200) [reply instagram.py]: Done

File: instagram.py (hunk 5 above)
Comment (priyank200) [reply instagram.py]: Done

**Review by sgdanawe — APPROVED** [review]

**Pull request merged**